src/api/info.rs

```diff
@@ -0,0 +1,16 @@
+use actix_web::HttpRequest;
+use actix_web::HttpResponse;
+use serde::Serialize;
+
+#[derive(Serialize)]
+struct Info {
+    version: &'static str,
+}
+
+const INFO: Info = Info {
+    version: env!("CARGO_PKG_VERSION"),
+};
+
+pub async fn information(_: HttpRequest) -> HttpResponse {
```

Comment:
Is the param necessary? `cargo run` doesn't seem to complain locally if I do:

```suggestion
pub async fn information() -> HttpResponse {
```